08-projects.Rmd

```diff
@@ -4815,53 +4815,74 @@ dbGetQuery(con, "SELECT * FROM battings LIMIT 5;")
+- Basic clauses: SELECT, ORDER BY, LIMIT, DESC, ASC, COUNT, WHERE, FROM, etc.
 - Showcase the ability to filter, alias, and write subqueries.
-- Perform grouping and aggregate data using group by and the following functions: count, max, sum, avg, like, having. Explain when to use having, and when to use where.
+- Perform grouping and aggregate data using group by and the following functions: COUNT, MAX, SUM, AVG, LIKE, HAVING. Explain when to use having, and when to use where.
 ```
 
 #### Dataset
 
-The following questions will use the `elections` database and the following database found in Scholar:
+The following questions will use the `elections` database. Similar to Project 10, this database is hosted on Scholar. Moreover, Question 1 also involves the following data files found in Scholar:
 
 `/class/datamine/data/election/itcontYYYY.txt` (for example, data for year 1980 would be `/class/datamine/data/electionitcont1980.txt`)
 
-A public sample of the data can be found [here](https://www.datadepot.rcac.purdue.edu/datamine/data/election/itcontYYYY.txt)
+A public sample of the data can be found [here](https://www.datadepot.rcac.purdue.edu/datamine/data/election/itcontYYYY.txt).
 
 Up until now, you've been working with a neatly organized database containing baseball data. As fantastic as this database is, it would be trivial to load up the entire database in R or Python and do your analysis using `merge`-like functions. Now, we are going to deal with a much larger set of data.
 
 #### Questions
 
-##### 1. Approximately how large was the lahman database (use the sqlite database in Scholar: `/class/datamine/data/lahman/lahman.db`)? Use UNIX utilities you've learned about this semester to write a line of code to return the amount of data (in MB) in the elections folder `/class/datamine/data/election/`. How much data (in MB) is there? 
+**Important note:** For this project you will need to connect to the database `elections` using the `RMariaDB` package in R. Include the following code chunk in the beginning of your RMarkdown file:
+
+````
+```{r setup-database-connection}
+library(RMariaDB)
+con <- dbConnect(RMariaDB::MariaDB(), 
+                 host="scholar-db.rcac.purdue.edu", 
+                 db="elections", 
+                 user="elections_user", 
+                 password="Dataelect!98")
+```
+````
+When a question involves SQL queries in this project, you may use a SQL code chunk (with `{sql}`), or an R code chunk (with `{r}`) and functions like `dbGetQuery` as you did in Project 10. Please refer to Question 5 in the [project template](https://raw.githubusercontent.com/TheDataMine/the-examples-book/master/files/project_template.Rmd) for examples.
+
+##### 1. Approximately how large was the lahman database (use the sqlite database in Scholar: `/class/datamine/data/lahman/lahman.db`)? Use UNIX utilities you've learned about this semester to write a line of code to return the size of that .db file (in MB). 
```

Comment:
I re-worded Question 1 to clarify the three tasks involved. However, I am not sure whether `##### text` would look identical to `**text**`...